symfony-cmf/routing-bundle/2.0/config/packages/cmf_routing_bundle.yaml

```diff
@@ -0,0 +1,14 @@
+# Read the documentation: https://symfony.com/doc/master/cmf/bundles/routing/configuration.html
+cmf_routing:
+    chain:
+        routers_by_id:
+            cmf_routing.dynamic_router: 200
+            router.default:             100
+    dynamic:
+        persistence:
+            phpcr:
+                enabled: true
+
+framework:
+    templating: { engines: ['twig'] }
```

Comment:
is that a good way to do it? I would be better to extend the original one from my POV. But we need that one, we got troubles on `make cache-warmup` else.

Reply:
In the https://github.com/symfony-cmf/routing-bundle/blob/master/composer.json file you should add symfony/templating to the `require` dependencies.

Reply:
@javiereguiluz did we get that one with an other component/bundle before? btw: it feels more like i need assets:
```
SYMFONY_ENDPOINT=https://symfony.sh/r/github.com/symfony/recipes-contrib/91 composer req symfony-cmf/routing-bundle
....
...
!!    [Symfony\Component\DependencyInjection\Exception\LogicException]          
!!    Asset support cannot be enabled as the Asset component is not installed.  
```

Reply:
`symfony/templating`, `asset` and `validator` is my current state of missing components :-)

Reply:
@javiereguiluz how shall i add it, when pushing the missing dependencies to master i won't ever get them here until we release 2.1 and i have no clue if adding dependencies can go onto a a minor release? @dbu @wouterJ?

Reply:
yes they can go in a minor version (even in a patch version). This is not a BC break, as composer handles it for the user.